src/tsserver/server.ts

```diff
@@ -484,8 +484,9 @@ namespace ts.server {
         private constructed: boolean | undefined;

```

Comment:
i think you can get rid of private variable and always use the local variable instead

Reply:
this variable is used in other method that cannot access to the local variable